local_costmap:
  global_frame: map
  robot_base_frame: base_link
  update_frequency: 1
  publish_frequency: 0.5
  static_map: false
  rolling_window: true
  width: 5
  height: 5
  resolution: 0.05
  transform_tolerance: 0.5
  cost_scaling_factor: 5
  # inflation_radius: 0.3
